### PR TITLE
bpo-36540 What's New section for PEP 570

### PR DESCRIPTION
**Pull request by gvanrossum**
@pablogsal You can either copy this into GH-12701 or land it after that has landed.

https://bugs.python.org/issue36540

### CODE REVIEW

**Review by vstinner — APPROVED** [review]
LGTM.

**Comment by pablogsal** [comment]
Thanks for doing this! I will land this after GH-12701 as this is more likely to have merge conflicts (as it has as the time of writing this) and I prefer not to rebase multiple times a PR that is already very big. Also, I will make a separate PR with all the doc changes that are more likely to need native speaker reviews.

**Comment by gvanrossum** [comment]
OK, fixed the merge conflict (of my own making). Are you gonna make alpha 4?

File: Doc/whatsnew/3.8.rst
Comment (pganssle) [inline Doc/whatsnew/3.8.rst]:
```suggestion
New syntax (``/``) has been added to indicate that some function parameters
```

"There is new syntax" also is fine grammatically, but usually the voice of these "What's new" entries seems to assume the context is that you are explaining what this PEP changes. "There is new syntax" feels every so slightly off because it de-emphasizes the change aspect of it.

I'll note that to me this is a very subtle difference and if I were not reading this specifically with giving notes on the wording in mind I would probably not have batted an eye, so if you prefer the original wording that's fine.

Comment (pganssle) [inline Doc/whatsnew/3.8.rst:89]: Maybe "PEP 570 adds new syntax..." instead of "There is new syntax"?

"There is new syntax" is also fine grammatically, but usually the voice of these "What's new" entries seems to assume the context is that you are explaining what this PEP changes. "There is new syntax" feels every so slightly off because it de-emphasizes the change aspect of it.

I'll note that to me this is a very subtle difference and if I were not reading this specifically with giving notes on the wording in mind I would probably not have batted an eye, so if you prefer the original wording that's fine.

**Comment by pablogsal** [comment]
> OK, fixed the merge conflict (of my own making). Are you gonna make alpha 4?

That's my intention but I am still waiting for a review from @serhiy-storchaka (Victor already approved). If we are very close to the date, I would like to merge it (maybe without the changes in inspect as Serhiy suggested) and then we can fix particular details afterwards.

**Review by pganssle — APPROVED** [review]
Other than my one *optional* style note, I see no problems with the grammar and style of this.

**Comment by gvanrossum** [comment]
I'll leave it up to Pablo.

File: Doc/whatsnew/3.8.rst
Comment (pablogsal) [reply Doc/whatsnew/3.8.rst:89]: I will leave  `There is new syntax` for symmetry with the assignment expressions paragraph.

**Pull request merged**
